src/app/docs/split-view/split-view-docs.component.html

```diff
@@ -8,9 +8,146 @@
+        <form
+          [formGroup]="splitViewDemoForm"
+          (ngSubmit)="onApprove()"
+        >
+          <sky-definition-list
+            labelWidth="150px"
+          >
+            <sky-definition-list-content>
+              <sky-definition-list-label>
+                Receipt amount
+              </sky-definition-list-label>
+              <sky-definition-list-value>
+                {{ activeRecord.amount }}
+              </sky-definition-list-value>
+            </sky-definition-list-content>
+            <sky-definition-list-content>
+              <sky-definition-list-label>
+                Date
+              </sky-definition-list-label>
+              <sky-definition-list-value>
+                {{ activeRecord.date }}
+              </sky-definition-list-value>
+            </sky-definition-list-content>
+            <sky-definition-list-content>
+              <sky-definition-list-label>
+                Vendor
+              </sky-definition-list-label>
+              <sky-definition-list-value>
+                {{ activeRecord.vendor }}
+              </sky-definition-list-value>
+            </sky-definition-list-content>
+            <sky-definition-list-content>
+              <sky-definition-list-label>
+                Receipt image
+              </sky-definition-list-label>
+              <sky-definition-list-value>
+                {{ activeRecord.receiptImage }}
+              </sky-definition-list-value>
+            </sky-definition-list-content>
+          </sky-definition-list>
+          <div
+            class="sky-form-group"
+          >
+            <label
+              class="sky-control-label"
+              for="approvedAmount"
+            >
+              Approved amount
+            </label>
+            <input
+              class="sky-form-control"
+              formControlName="approvedAmount"
+              name="approvedAmount"
+              type="text"
+            />
+          </div>
+
+          <div
+            class="sky-form-group"
+          >
+            <label
+              class="sky-control-label"
+              for="comments"
+            >
+              Comments
+            </label>
+            <textarea
+              class="sky-form-control"
+              formControlName="comments"
+              name="comments"
+            ></textarea>
+          </div>
+        </form>
+      </sky-split-view-workspace-content>
+      <sky-split-view-workspace-footer>
+        <sky-summary-action-bar
+          id="summary-action-bar"
+        >
+          <sky-summary-action-bar-actions>
+            <sky-summary-action-bar-primary-action
+              (actionClick)="onApprove()"
+            >
+              Approve expense
+            </sky-summary-action-bar-primary-action>
+            <sky-summary-action-bar-secondary-actions>
+              <sky-summary-action-bar-secondary-action
+                (actionClick)="onDeny()"
+              >
+                Deny request
```

Comment:
When I view the demo in the PR preview link, this shows up as a button with an ellipses icon and just the one item in the dropdown. In the live site, it's just a secondary button. Is this an error? I assume so because I don't think our Design guidance would be to have a single item in a dropdown when there is plenty of room for the label to appear in the button instead of an ellipses. ... I assume we just need to get rid of the wrapping `sky-summary-action-bar-secondary-actions` element. Do we really need that?

Also, I assume that the object of the two buttons is the same. In the current demo and in this new version, we say `Approve expense` and `Deny request`. It seems we should stick to `expense` or `request` in both cases instead of using synonyms. This would be more consistent with our guidance elsewhere on labels and UI text.

Reply:
Hmmm. Going to look into the dropdown. Appears to be something strange with the component and may be a bug in the overall component. That should only be happening on very small screen/split view sizes.

Reply:
I have changed the wording to be `expense` for both buttons.

Reply:
@blackbaud-johnly the dropdown issue here is in fact a bug with the component. Going to fix that in a separate PR so that it doesn't cause release confusion.

Reply:
You can notice that if you barely resize the split view's inner window the dropdown turns into a button.

Reply:
https://github.com/blackbaud/skyux-split-view/issues/42

Reply:
That's weird that it turns into a button after resizing. ... The label changes look good, so I'll close this conversation given that we have the separate PR to fix the bug. Thanks!